normandy/settings.py

```diff
@@ -255,6 +255,7 @@ def LOGGING(self):
     ACTION_IMPLEMENTATION_CACHE_TIME = values.IntegerValue(60 * 60 * 24 * 365)
     NUM_PROXIES = values.IntegerValue(0)
     API_CACHE_TIME = values.IntegerValue(30)
+    LOG_OSCPU_RATE = values.FloatValue(0.001)
```

Comment:
New settings need to go [in the docs](https://github.com/mozilla/normandy/blob/73bd7183f1c4c692308ad866397dc915350cbf42/docs/ops/config.rst).